tests/filtered_table_model_tester.ts

```diff
@@ -4,7 +4,7 @@ import { AddRowOperation, ArrayTableModel, FilteredTableModel, Operation,
   from '../source';
 
 class MockPredicate {
-  public applyPredicate(row: number, model: TableModel): boolean {
+  public applyPredicate(model: TableModel, row: number): boolean {
```

Comment:
Remove this class, you can define a function if you'd like but no need to have it be part of a class.